tests/Fixtures/Integration/misc/issue_6492.test

```diff
@@ -0,0 +1,26 @@
+--TEST--
+PHPStan comment after anonymous class opening brace must stay.
```

Comment:
Please, update the description or make the comments PHPStan's ones, so the test is consistent.

Reply:
addressed

Reply:
done

Reply:
👍, done